substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/ClassEntry.java

```diff
@@ -369,4 +374,9 @@ public MethodEntry ensureMethodEntryForDebugRangeInfo(DebugRangeInfo debugRangeI
     public List<MethodEntry> getMethods() {
         return methods;
     }
+
+    public String getSimpleClassName() {
```

Comment:
This seems to be necessary solely to determine whether a method is a constructor. I think it would be cleaner to introduce an API than to rely on a naming convention, say `DebugMethodInfo.isConstructor`, which could also be done as part of #4333.

Reply:
Done.